Forms/DateOfBirthUtils.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using UnityEngine;
+
+namespace DUCK.Forms
+{
+	public static class DateOfBirthUtils
+	{
+		public static int[] Days { get; private set; }
+		public static int[] Years { get; private set; }
+		public static string[] Months { get; private set; }
+
+		public enum MonthType
```

Comment:
Surely just 'Month'? A month isn't a Type of anything

Reply:
Doesn't this already exist in System.DateTime or are you deliberately avoiding using System.*?

Reply:
Agreed, can be `Month`, yes, its purposely not using DateTim as DateTime will throw if the values are not valid, this validation will occur later with Validators.